components/list/list.js

```diff
@@ -111,7 +112,7 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 	render() {
 		const role = !this.grid ? 'list' : 'application';
 		return html`
-			<slot name="header"></slot>
+			<section aria-label="${this.localize('components.list.toolbar')}"><slot name="header"></slot></section>
```

Comment:
@eKoopmans @dbatiste I figured it made sense to put this here rather than inside `selection-header`, since that seemed a bit more general.  But depending on how the table header/toolbar is going to work, maybe it makes more sense to be lower?  `d2l-lsit-header` itself doesn't `render` anything, so I'd have to expose a way for it to set the text.

Reply:
Yeah my thinking was the alternative would be to wrap [this](https://github.com/BrightspaceUI/core/blob/main/components/selection/selection-header.js#L132-L145) in a `section` and call `_getSelectionHeaderAriaLabel` or whatever to get the `aria-label`, which `d2l-list-header` would override.  Similar to how it gets the container classes right now.  But it did feel more complicated and I'd want to do more regression testing if I'm placing the section inside like that

Reply:
Hey sorry, just getting to this all now.

I personally like replacing (not wrapping) the [selection-header-container div](https://github.com/BrightspaceUI/core/blob/main/components/selection/selection-header.js#L132-L145) with a `section`, and having the aria-label overrideable.

Reply:
Another option - we could have the selection-header itself be the region, by doing something like:

```js
static get properties() {
    return {
        // ...
       role: { type: String, reflect: true },
       ariaLabel: { type: String, attribute: 'aria-label', reflect: true },
}

constructor() {
    super();
    this.role = 'region';
    this.ariaLabel = this.localize('components.selection.toolbar');
}
}
```

Reply:
Hmm yeah, the WCAG docs recommend `section` over using the `role` directly but I think this is a case where it would make a lot of sense - we don't _want_ another element that's messing with the sticky positioning.  We'd have to recalculate the langterm on language change if we did the above, or we could have the consumer pass it in.

@eKoopmans Do you know yet if `table-wrapper` will have a slot for the table header/toolbar?  As in, will _we_ be rendering the `table-header` and can pass in the langterm for the consumer (like list does), or will every consumer be rendering it themselves? I think that would make the decision on who passes/sets the aria-langterm

Reply:
The plan is to use the same pattern as list, with a slot for the table-header.

Reply:
Ok, here's what it looks like placed inside `d2l-selection-header`: https://github.com/BrightspaceUI/core/pull/3040
I did end up opening a second PR to make it easier to compare

Reply:
Seems good to me, and probably the preferred approach if it avoids issues with sticky position.